src/ResultsActivity/ResultsActivity.js

```diff
@@ -177,6 +184,7 @@ class ResultsActivity extends Component {
       const lockedCases = attendanceInfo.filter(activity => activity.attendanceInfo.locked === true)
 
       if (
+        isAfterToday(date) ||
```

Comment:
worth adding test for this scenario in ResultsActivity.test.js

Reply:
Good thinking.  I've added a few more tests to ResultsActivity around the date/button visibility.